create_test_data.py

```diff
@@ -0,0 +1,83 @@
+"""
+
+This is a script for generating test holdshelf events in the local psql db.
+
+Usage:
+    AWS_PROFILE=nypl-digital-dev ENVIRONMENT=development \
+        python create_test_data.py
+
+"""
+
+import os
+import datetime
+
+from nypl_py_utils import PostgreSQLClient, PostgreSQLClientError
+from nypl_py_utils.functions.config_helper import load_env_file
+
+HOLDS = [
+    [1, 10001, 'i', 'YYYY-MM-DDT08:11:01', 'mal'],
+    [2, 10002, 'i', 'YYYY-MM-DDT08:31:01', 'mal']
+]
+
+ITEMS = [
+    (10001, 101, 'maf92'),
+    (10002, 102, 'maf92')
+]
+
+
+class CreateTestData:
+
+    def connect(self):
+        load_env_file(os.environ['ENVIRONMENT'], 'config/development.yaml')
+        self.sierra_client = PostgreSQLClient(
+            '127.0.0.1',
+            os.environ['SIERRA_DB_PORT'],
+            os.environ['SIERRA_DB_NAME'],
+            os.environ['SIERRA_DB_USER'],
+            os.environ['SIERRA_DB_PASSWORD']
+        )
+        self.sierra_client.connect()
+
+    def disconnect(self):
+        self.sierra_client.close_connection()
+
+    def insert_data(self, table, row):
+        values_str = ', '.join(['%s' for n in range(len(row))])
```

Comment:
I think `['%s' for n in range(len(row))]` will be [0, 1, 2, 3, ...] up to the length of the row rather than the actual values of that row

Reply:
Ah I see